upgrade.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
+#
+# Simple script for upgrading AppScale to one of release versions.
+# Author: AppScale Team <support@appscale.com>
+
+set -e
+
+APPSCALE_REPO="git://github.com/AppScale/appscale.git"
+APPSCALE_TOOLS_REPO="git://github.com/AppScale/appscale-tools.git"
+AGENTS_REPO="git://github.com/AppScale/appscale-agents.git"
+THIRDPARTIES_REPO="git://github.com/AppScale/appscale-thirdparties.git"
+GIT_TAG="last"
+UNIT_TEST="N"
+
+usage() {
+    echo "Usage: ${0} [--tag <git-tag>] [-t]"
+    echo
+    echo "Options:"
+    echo "   --tag <git-tag>    Git tag (e.g.: 3.7.2) to upgrade to."
+    echo "                      Default: '${GIT_TAG}' (use the latest release)."
+    echo "   -t                 Run unit tests"
+    exit 1
+}
+
+version_ge() {
+    test "$(printf '%s\n' "$@" | sort -V | tail -1)" = "$1"
+}
+
+
+echo -n "Checking to make sure you are root..."
+if [ "$(id -u)" != "0" ]; then
```

Comment:
In the above script you used if then else on a single line for parameter checking: perhaps you should do the same here for consistency?

Reply:
I used short syntax for parsing argument as it was used in bootstrap script. In bootstrap script we have many arguments and full form made it hard to read parsing of  10 args.
I'd prefer to use multi-line syntax in other places.